host/maps.c

```diff
@@ -103,7 +103,7 @@ int myst_maps_load(myst_maps_t** maps_out)
 
         n = sscanf(
             line,
-            "%lx-%lx %c%c%c%c %lu %u:%u %lu %s",
+            "%lx-%lx %c%c%c%c %lu %u:%u %lu %4096s",
```

Comment:
where does this 4096 come from

Reply:
The max size for path is 4096 bytes

Reply:
> The path variable is of 4096 bytes in size

Is there a #define we can use instead?

Reply:
Hmm? I wish there were a way without hardcoding the path length.

Reply:
Okay, I verified that the following works:

```
        int path_offset;

        n = sscanf(
            line,
            "%lx-%lx %c%c%c%c %lu %u:%u %lu %n",
            &start,
            &end,
            &r,
            &w,
            &x,
            &f,
            &o,
            &minor,
            &major,
            &inode,
            &path_offset);

        if (n < 9)
        {
            ret = -ENOSYS;
            goto done;
        }

        *path = '\0';
        strncat(path, &line[path_offset], PATH_MAX);
```

Reply:
Also, you probably want to check whether ``streln(*line[path_offset]) is bigger than ``PATH_MAX``.